### ISSUE TITLE
Add .NET 7.0 support

### ISSUE THREAD

**Issue by Illia-M** [lucaslorentz/minicover#161]
Add support for .NET 7.0 apps #160

**Comment by Illia-M** [comment]
@ffMathy when you plan publish new package version?

**Comment by ffMathy** [comment]
Yeah I really need to automate this part 😅 haven't had time recently.

**Comment by csharp4fun1** [comment]
I had to switch to dotnet-coverage because I can't wait any longer to have statistics again (need that for reports at work). If this project should be successful that time invest has to be done.

**Comment by lucaslorentz** [comment]
@csharp4fun1 Thanks for the feedback. Honestly I think it's a good move to switch to coverage tools maintained by Microsoft. But I'm doing a new release anyway: https://github.com/lucaslorentz/minicover/releases/tag/v3.6.0

I have abandoned this project some time ago, and @ffMathy was kindly maintaining it for a bit longer. Thanks again @ffMathy 

@csharp4fun1 Does Minicover brings any extra benefits to you compared to dotnet-coverage or coverlet? That input would be appreciated and would help me understand if it is worth actively maintaining this project.

**Comment by csharp4fun1** [comment]
Let's say like that: some time ago (I think one and a half years ago), dotnet-coverage wasn't available on linux. That was the main reason to go to Minicover. It brings everything we need, creates cobertura report files. 

I think it is pretty good to have tools as alternative to the official ones. It could be that Microsoft has bad ideas, stops maintaining their tools or other reasons to go with other open source alternatives. So I think it is important to not let it die. I know it is a lot of extra work, @lucaslorentz and @ffMathy you are doing a good job, maybe also other maintainers can be found for future adaptions. I would like to do, but sorry - currently with the second baby a lot extra work to do anyways ^^

My personal reason to switch was another:
In the middle management where I'm situated at work I have to report coverage for the projects of my teams. As we didn't want to wait for a tool (Minicover) to switch to .NET 7 it brought me into a bad situation. For November statistics I created branches for each project where I have to do reporting on and switched back to .NET 6. There were not too many changes I had to make (e.g. reverting raw string literals), nevertheless it was a lot of extra work on the weekends for me to get my numbers. For the December statistics I didn't want to do that effort again and so I switched to dotnet-coverage.
